packages/@aws-cdk/aws-eks/lib/kubectl-provider.ts

```diff
@@ -67,6 +67,7 @@ export class KubectlProvider extends NestedStack {
     }
 
     const layer = cluster.kubectlLayer ?? getOrCreateKubectlLayer(this);
+    const memorySize = cluster.kubectlMemory ? cluster.kubectlMemory.toMebibytes() : 1024;
```

Comment:
I don't think we should change the default size, it has cost implications and seems like 256 is enough for most manifests.

Reply:
The monthly cost for 500 requests of 10min is:
- $1.25US for 256MiB
- $2.50 for 512MiB
- $5US for 1024MiB

I think 1GiB should be fine.

Reply:
I'm ok with saying we are productionizing the configuration, but let's make sure we mention the value change in the changelog.